docs/resources/client/models/index.md

```diff
@@ -235,30 +235,32 @@ In contrast, inside a `multipart` block, elements are combined depending on the
 Some blocks, such as grass or leaves, change their texture color based on their location and/or properties. [Model elements][elements] can specify a tint index on their faces, which will allow a color handler to handle the respective faces. The code side of things works through two events, one for block color handlers and one for item color handlers. They both work pretty similar, so let's have a look at a block handler first:
 
 ```java
+// Client-side mod bus event handler
 @SubscribeEvent
 public static void registerBlockColorHandlers(RegisterColorHandlersEvent.Block event) {
     // Parameters are the block's state, the level the block is in, the block's position, and the tint index.
     // The level and position may be null.
     event.register((state, level, pos, tintIndex) -> {
-            // Replace with your own calculation. See the BlockColors class for vanilla references.
-            // All vanilla uses assume alpha 255 (= 1f), but modded consumers may also account
-            // for alpha values specified here. Generally, if the tint index is -1,
-            // it means that no tinting should take place and a default value should be used instead.
-            return 0xFFFFFF;
+        // Replace with your own calculation. See the BlockColors class for vanilla references.
+        // All vanilla uses assume alpha 255 (= 1f), but modded consumers may also account
```

Comment:
This is not correct anymore, block and item colors now respect the alpha component of the returned value (assuming the render type of the block/item can deal with it, of course)